.idea/.gitignore

```
@@ -0,0 +1,2 @@
+# Default ignored files
+/workspace.xml
```

I also use intelliJ let's also add `.idea` folder to gitignore :) 
```suggestion
/workspace.xml
.idea
```

---

Ha yeh, I was going to do that actually. It's annoying!